server/channels/api4/user_test.go

```diff
@@ -1564,15 +1601,17 @@ func TestAutocompleteUsersInChannel(t *testing.T) {
 
 	t.Run("Check OutOfChannel results with/without VIEW_MEMBERS permissions", func(t *testing.T) {
```

Comment:
The test is just broken. I tried to fix it, but it seems like the logic it tests is just invalid. Let's disable it for now.
```suggestion
	t.Run("Check OutOfChannel results with/without VIEW_MEMBERS permissions", func(t *testing.T) {
		t.Skip("https://mattermost.atlassian.net/browse/MM-61041")
```

Reply:
The latter part adds the correct license for the test.